contrib/hashicorp/consul/example_test.go

```diff
@@ -0,0 +1,46 @@
+package consul
+
+import (
+	"context"
+	"fmt"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	consul "github.com/hashicorp/consul/api"
+)
+
+func Example() {
```

Comment:
If we leave some text above this example, it will look nice in `godoc` when compiled into documentation, e.g.:
```go
// Here's an example illustrating a simple use case for interacting with consul with tracing enabled.
```

Reply:
Done